File: eSpaCenter.Models/NarudzbaProizvodi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eSpaCenter.Models
{
    public class NarudzbaProizvodi
    {
        public int NarudzbaProizvodiID { get; set; }
        public int ProizvodID { get; set; }
        public int NarudzbaID { get; set; }
        public int Kolicina { get; set; }

        public virtual Proizvod Proizvod { get; set; }
    }
}

File: UI/espacenter_admin/lib/models/narudzba_proizvodi.dart
import 'package:espacenter_admin/models/proizvod.dart';
import 'package:json_annotation/json_annotation.dart';

part 'narudzba_proizvodi.g.dart';

@JsonSerializable()
class NarudzbaProizvodi{
  int? narudzbaProizvodiID;
  int? proizvodID;
  int? narudzbaID;
  int? kolicina;
  Proizvod? proizvod;

  NarudzbaProizvodi(this.narudzbaProizvodiID,this.proizvodID,this.narudzbaID,this.kolicina,this.proizvod);

  factory NarudzbaProizvodi.fromJson(Map<String, dynamic> json) => _$NarudzbaProizvodiFromJson(json);

    Map<String, dynamic> toJson() => _$NarudzbaProizvodiToJson(this);
}